jobs/e2e-test.groovy

```diff
@@ -290,6 +293,28 @@ def runTestServer() {
          runSmokeTestsArgs += ["--skip-tests=${params.SKIP_TESTS}"];
       }
 
+      // Determine if we actually need to run any tests at all
+      // tests = sh(script: "testing/runtests_server.py -n ${exec.shellEscapeList(runSmokeTestsArgs)} . ", returnStdout: true);
+      tests = exec.outputOf(["testing/runtests_server.py", "-n", runSmokeTestsArgs, "."].flatten());
```

Comment:
Oh!  I see how you did that.  You don't need the `flatten()`, you can just do:
```
tests = exec.outputOf(["testing/runtests_server.py", "-n"] + runSmokeTestsArgs + ["."])
```

Reply:
OK, agreed that 7 seconds is not the end of the world. :-)  But I still think it would appropriate to add a TODO to refactor things somehow so we don't need this extra call to runtests_server.py.

Or maybe do what we do for the unit-tests, and actually use this list as input to the next runtests_server.py call at line 333!  But that can be a TODO as well, since I don't want to make you go through another round of testing.